modules/table/src/test/java/org/apache/ignite/internal/table/type/NumericTypesSerializerTest.java

```diff
@@ -0,0 +1,203 @@
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.table.type;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+import org.apache.ignite.internal.schema.Column;
+import org.apache.ignite.internal.schema.InvalidTypeException;
+import org.apache.ignite.internal.schema.NativeTypes;
+import org.apache.ignite.internal.schema.SchemaDescriptor;
+import org.apache.ignite.internal.schema.marshaller.TupleMarshaller;
+import org.apache.ignite.internal.schema.registry.SchemaRegistryImpl;
+import org.apache.ignite.internal.schema.row.Row;
+import org.apache.ignite.internal.table.TupleBuilderImpl;
+import org.apache.ignite.internal.table.TupleMarshallerImpl;
+import org.apache.ignite.internal.util.Pair;
+import org.apache.ignite.schema.ColumnType;
+import org.apache.ignite.table.Tuple;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/** */
+public class NumericTypesSerializerTest {
+    /** Random. */
+    private Random rnd = new Random();
+
+    /** Tuple marshaller. */
+    private TupleMarshaller marshaller;
+
+    /** Schema descriptor. */
+    private SchemaDescriptor schema;
+
+    /**
+     * @return List of BigInteger pairs for test.
+     */
+    private static List<Pair<BigInteger, BigInteger>> numbers() {
+        return Arrays.asList(
+            new Pair<>(BigInteger.valueOf(10L), BigInteger.valueOf(10)),
+            new Pair<>(BigInteger.valueOf(-10L), BigInteger.valueOf(-10)),
+            new Pair<>(new BigInteger("10"), BigInteger.valueOf(10)),
+            new Pair<>(new BigInteger("1000").divide(BigInteger.TEN), BigInteger.valueOf(10).multiply(BigInteger.TEN)),
+            new Pair<>(new BigInteger("999999999"), BigInteger.valueOf(999999999L)),
+            new Pair<>(new BigInteger("+999999999"), BigInteger.valueOf(999999999L)),
+            new Pair<>(new BigInteger("-999999999"), BigInteger.valueOf(-999999999L))
+        );
+    }
+
+    @BeforeEach
+    public void setup() {
+        long seed = System.currentTimeMillis();
+
+        rnd = new Random(seed);
+
+        marshaller = new TupleMarshallerImpl(new SchemaRegistryImpl(v -> null) {
```

Comment:
We already have DummySchemaManagerImpl that can be reused.
Maybe it make sense to rename it DummySchemaRegistry